README.md

```diff
@@ -5,46 +5,58 @@ This readme assumes that you have the following programmes installed:
 - IDE e.g. VS Code
 - MongoDB Compass
 - Node (version 19.0.1)
+- Git
 
-This ReadMe has been made in a .md format so it can be opened in your IDE previewer for easier reading.
+**PLEASE NOTE:**
+
+This ReadMe was written using the built-in terminal in VS Code. It was tested using this terminal on both MacOS and WindowsOS. 
 
 --- 
 ## Setup & Install
-Navigate to the folder where you want the application to be stored on your computer   
+
+1. Open a terminal on MacOS or PowerShell on Windows.  
+
+2. Navigate to the folder where you want the application to be stored on your computer.  
 > i.e. `$ cd Desktop/Webapp`  
 
-Clone the repo into this folder from this link --> (https://github.com/OrrionB/UniWebapp.git) 
+3. Clone the repo into this folder from this link --> (https://github.com/OrrionB/UniWebapp.git) 
 > i.e `$ git clone https://github.com/OrrionB/UniWebapp.git`  
   
-Install the dependencies  
-1. Open 2 terminal windows.
-2. In the first termnial window navigate to the folder where you installed the webapp and run the command `npm install`.  
+ 4. Open the folder in VS Code. 
+  
+4. Install the dependencies  
+- Open 2 terminals in VSCode  
+- In the first termnial window navigate to the folder where you installed the webapp and run the command `npm install`.  
```

Comment:
I'd change this to '...cd into the root of the webapp and run...' rather than navigate into the folder. It's just a bit clearer